components/cd-facets/cd-facets.css

```diff
@@ -0,0 +1,146 @@
+.block-facets {
```

Comment:
I'm assuming faceted search markup is difficult and/or annoying to override, and that's why this component doesn't use `cd-facets` as a namespace for its classes?

Reply:
This is so Drupal-specific it almost shouldn't be part of the components as there will likely never be a plain HTML version of this. 
I'd consider this just underdeveloped so I can't comment specifically on whether overriding is easy or not. When I am updating the implementations, I will have to revisit this since many use Facets. I will look at it again from that perspective.